Porting/bisect-runner.pl

```diff
@@ -958,6 +958,45 @@ =head2 When did perl start failing to build on a certain platform using C<g++> a
 
 =back
 
+=head2 When did a test file start to emit warnings?
+
+=over 4
+
+=item * Problem
+
+While F<make test> was running, warnings not previous seen were being emitted
```

Comment:
`C<make test>`; "previously"

Reply:
Thanks.  I reworked the "Problem" part a bit more; rebroke the `-e` command; and was able to remove the line from the POD issues database.